sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/SqlAnalyzer.java

```diff
@@ -115,6 +116,22 @@ static boolean isEndOfInput(ParseResumeLocation parseResumeLocation) {
         >= parseResumeLocation.getInput().getBytes(UTF_8).length;
   }
 
+  static String getOptionStringValue(
```

Comment:
nit: This method appears to be used exactly once in another file. It should go right next to the method that calls it.